src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -154,7 +154,7 @@ private void OnAnalysisRequested(object sender, AnalysisRequestEventArgs args)
             if (detectedLanguages.Any() && analyzerController.IsAnalysisSupported(detectedLanguages))
             {
                 // Multiple views could have that buffer open simultaneously, so only create one instance of the tracker.
-                var issueTracker = buffer.Properties.GetOrCreateSingletonProperty(typeof(TextBufferIssueTracker),
+                var issueTracker = buffer.Properties.GetOrCreateSingletonProperty(typeof(IIssueTracker),
```

Comment:
The `Type` is being used as a key for the singleton property. I've checked that the behaviour is the same with this change.